### ISSUE TITLE
Audiogram correction equalizer for hard hearing people on their personal computer  

### ISSUE THREAD

**Issue by PietZ1959** [jaakkopasanen/AutoEq#516]
Dear Jaakko, my name is Piet van der Zanden from the Netherlands. I focus at readability and ergonomic qualities in education spaces, see https://www.linkedin.com/in/pietvanderzanden/. I am (a bit) hard hearing and have problems to follow online hybrid meetings now they are everywhere these days since the pandemic.

With inclusion in mind I was looking for a free equalizer for my laptop to correct the incoming audio from Teams and Zoom sessions since I do wear headphones with such online sessions but not my hearing aid. Would it be possible to use your AutoEq for correcting my hearing i.e. correcting the audiogram of my ears? 
I imagine that more people would like that to have a better inclusion for the hard hearing when they have to meet online :-)

Warm wishes, Piet van der Zanden
![Piet's Audiogram](https://user-images.githubusercontent.com/112757628/188268311-015e063f-a5f5-42b1-8158-a742ca440058.jpg)

**Comment by jaakkopasanen** [comment]
Interesting use case

In theory this is possible and AutoEq already has everything needed. [WebPlotDigitizer](https://apps.automeris.io/wpd/) can be used to get the numerical data from the audiograms. However, I'm not convinced that this could be done in practice as I've seen filters with gain more than 20 dB or so aren't really possible with many equalizers. Also there's a question if the audiogram should be compensated as is. Your brains have adapted to the hearing loss to a degree so fully compensating for it in equalizer might not be the way to go.

I'm going to close this issue as there's nothing more I can do in AutoEq to support this use case. The problems that might prevent this from being possible are outside of AutoEq. You might want to ask in [/r/oratory1990](https://www.reddit.com/r/oratory1990/) if headphone eq is a suitable tool for this in the first place. My knowledge of the topic falls short here I'm afraid.

**Comment by PietZ1959** [comment]
Dear Jaakko, thank you very much for the honest answer. I really do appreciate it :-)
I guess you are right in both ways: my hearing is indeed adapted over time and not directly proportional with the frequency bands of the hearing aid, and I like to correct up to 80 dB to come close to it. I have seen a few equalizers up to 60 dB that are free to be used for non commercial use. So, I will look further.
My idea was to "calibrate" the equalizer to personal (left and right headphone) settings based on more than the 8 standard bands of hearing aids with the help of pink noise and STIPA reference signals. And to switch it off when presenting to others.
Thanks again, warm wishes, Piet.

**Comment by PietZ1959** [comment]
Dear Jakob, sorry for my late response, I just came back from my holidays. Thanks for your suggestion to use EqualizerAPO, I will look into it after I have worked away my backlog :-)

Indeed is overcoming such enormous differences in sound levels not advisable, but I was exploring several ways to get some grip in making the online hearing a bit better. I just do not know yet were the right balance or solution will be. 
In a few weeks time I will meet the hospital ear specialist and shall discuss why we want to try to level up only my right ear with a hearing aid (which will never come 100 % of speech intelligibility) up to the hearing of my left ear, instead of just ignoring my right ear and put a hearing aid to my left ear only. Amplifying that one up to a level of about 20 tot 25 dB will probably deliver up to full hearing so I can finally hear the unvoiced consonants in order to better understand (foreign) conversations. Next to that I hope that I can find time to do measurements before the end of year when a meeting takes place in a "half dead" room versus a "reverberating" room while listening at the far end.

Thanks again, Piet
